resources/cygnus-translators/cygnus_translator_pre0.10.0_to_0.10.0_ckan.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ba085d5de84bd56ba290fcb08b9a9a7ff155e4f8